.github/ISSUE_TEMPLATE/good_first_issue.yaml

```diff
@@ -10,8 +10,9 @@ body:
       label: Summary
       description: |
         A summary of the issue.
-  - type: markdown
+  - type: textarea
```

Comment:
With `markdown`, the submitted issue doens't contain the notes section:

https://github.com/mlflow/mlflow/issues/9327

<img width="878" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/006cbe54-76ad-4492-b381-27262d698655">